plenum/common/func_utils.py

```diff
@@ -0,0 +1,28 @@
+
+
+def flatten(some_list):
+    """
+    [[a,b],[c,d]] => [a,b,c,d]
+    """
+    for inner_list in some_list:
+        for item in inner_list:
+            yield item
+
+
+def decons(some_list):
+    """
+    [a,b,c,d] => (a, [b,c,d])
+    """
+    if len(some_list) < 2:
+        return None
+    return some_list[0], some_list[1:]
+
+
+def count(some_list):
```

Comment:
Updating example.
Yeah looks same, but I leave it since it requires no additional object creation.